packages/src/lib/footer/footer.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectionStrategy, Component, input } from '@angular/core';
+
+import { FooterCopyrightComponent } from './footer-copyright/footer-copyright.component';
+import { FooterLinksComponent } from './footer-links/footer-links.component';
+import { FooterNavComponent } from './footer-nav/footer-nav.component';
+import { IFooterConfig } from './footer.interface';
+
+@Component({
+  selector: 'sdg-footer',
+  standalone: true,
+  imports: [FooterNavComponent, FooterLinksComponent, FooterCopyrightComponent],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
+})
+export class FooterComponent {
+  footer = input.required<IFooterConfig>();
```

Comment:
Subjectif comme commentaire mais j'aurais tendance à éclater ce input en plusieurs pour laisser le soin au assembleur de le paramétrer plus facilement via le template sinon avec un objet c'est seulement au niveau d'une config. J'aime bien la définition de l'API du HeaderComponent...
```suggestion
  title = input.required<string>();
  logo = input<Logo>();
  sections = input<Sections>();
  links = input<Sections>();
  copyright = input<Copyright>();
```

Il y a aussi les noms `sections` versus `links`, je proposerais `siteMap` pour le recap de la structure de navigation  et `externalLinks` pour les autres dans la section blanche?